apps/trivium/benches/casting.rs

```diff
@@ -0,0 +1,81 @@
+use tfhe::shortint::prelude::*;
+
```

Comment:
we may want to bench the casting in shortint actually

Reply:
Do you suggest moving this file in `tfhe/benches/shortint` ?